doc/user_download_and_install_windows.rst.template

```diff
@@ -0,0 +1,355 @@
+"Programs and Features" and click "Turn Windows Features On or Off". Check the box
+"Windows Subsystem for Linux (Beta)". Restart Computer. Search "Bash" to open Ubuntu 14.04
+subsystem terminal.
+
+
+Download the code
+=================
+
+The latest stable source code release of HORTON can be downloaded here:
+
+    https://github.com/theochem/horton/releases/download/2.0.1/horton-2.0.1.tar.gz
+
+Choose a suitable directory, e.g. ``~/build``, download and unpack the archive
+
+.. code-block:: bash
+
+    mkdir -p ~/build
+    cd ~/build
+    curl -kLO https://github.com/theochem/horton/releases/download/2.0.1/horton-2.0.1.tar.gz
+    tar -xvzf horton-2.0.1.tar.gz
+    cd horton-2.0.1
+
+For Developers
+--------------
+If you have git installed,
+
+.. code-block:: bash
+
+    git clone https://github.com/theochem/horton.git
+
+You may also want to checkout the HORTON release `2.0.0`. It seems to be more stable on Windows.
+
+.. code-block:: bash
+
+    git checkout 2.0.0
+
+
+Check Ubuntu Version (Just In Case)
+===================================
+On bash,
+
+.. code-block:: bash
+
+  lsb_release -a
+
+It should be `14.04.4 LTS`, but other versions may be available.
+
+
+Dependencies for building, installing and testing HORTON
+========================================================
+
+In order to compile and test HORTON, you need to install relatively recent
+versions of the following programs/libraries:
+
+${dependencies_rst}
+
+
+Installing dependencies with a package manager
+----------------------------------------------
+
+With Ubuntu, most of these dependencies can be installed with a `apt-get`:
+
+* **Ubuntu Linux 16.04**:
+
+  .. code-block:: bash
+
+    sudo apt-get install ${dependencies_ubuntu_16_deb}
+
+  Note that Ubuntu 16.04 does not have up-to-date packages for all dependencies. LibInt2
+  must be compiled manually as explained in :ref:`linux_manual_dependency_install`. (The
+  beta version of LibInt2 that comes with Ubuntu 16.04 is compiled with a too low
+  LIBINT2_MAX_AM_ERI value.)
+
+* **Ubuntu Linux 15.04 and 14.04**:
+
+  .. code-block:: bash
+
+    sudo apt-get install ${dependencies_ubuntu_15_deb}
+    pip install --user --upgrade ${dependencies_ubuntu_15_pip}
+
+  Note that Ubuntu 15.04 and 14.04 do not have up-to-date packages for all dependencies.
+  Hence, some are installed with PIP. LibInt2 must be compiled manually as explained in
+  :ref:`linux_manual_dependency_install`.
+
+* **Precautionary Measures:
+  Those of you that are installing HORTON as a dependency for other possibly "top
+  secret" stuff (you know who you are) should also upgrade `scipy` and `matplotlib`.
+
+  Scipy is quite simple to upgrade:
+  .. code-block:: bash
+
+    pip install --user --upgrade scipy
+
+  Matplotlib requires a little more:
+  .. code-block:: bash
+
+    sudo apt-get install libfreetype6-dev libpng pkg-config
+    ln -s /usr/include/freetype2/ft2build.h /usr/include/
```

Comment:
The right way to do this is by setting an INCLUDE_PATH for freetype2 in pipI think. This is a somewhat dangerous operation in the long run.